wicker/core/storage.py

```diff
@@ -32,6 +33,22 @@
 class AbstractDataStorage(ABC):
     """Abstract data storage class that implements required methods for Column Bytes File Cache"""
 
+    def __init__(self):
```

Comment:
@pickles-bread-and-butter this is what I want to do:
- I want to nuke the `_download` function from `AbstractDataStorage`.
- In `FileSystemDataStorage` have `fetch_file` call `_safe_file_download` directly.
- In `S3DataStorage` nuke `_download` and restore the previous `fetch_file` function completely.

There will be some code duplication between `_safe_file_download` and `fetch_file` in `S3DataStorage`, but they have pretty different use cases and setups so I think that's fine.

Reply:
Done, and makes sense